src/overlays/actors/ovl_En_Bb/z_en_bb.c

```diff
@@ -631,38 +615,69 @@ s32 EnBb_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* po
     return false;
 }
 
+/**
+ * This maps a given limb based on its limbIndex to its appropriate index
+ * in the bodyPartsPos/Velocity arrays. An index of -1 indicates that the
+ * limb is not part of the bodyParts arrays.
```

Comment:
```suggestion
 * in the bodyPartsPos/Velocity arrays.
```
Not sure this is needed anymore with the NONE enum?
Looks to appear in a couple other places as well.